pkgs/development/libraries/incrtcl/default.nix

```diff
@@ -22,9 +22,10 @@ stdenv.mkDerivation rec {
     libPrefix = "itcl3.4";
   };
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = http://incrtcl.sourceforge.net/;
     description = "Object Oriented Enhancements for Tcl/Tk";
-    platforms = stdenv.lib.platforms.unix;
+    platforms = platforms.unix;
+    license = licenses.bsd0;
```

Comment:
The wording in [license.terms](https://core.tcl.tk/itcl/artifact/4118a6307945b2f9) actually matches the [TCL/TK license](https://spdx.org/licenses/TCL.html) (which is BSD-style) and we have `licenses.tcltk`

Reply:
Indeed! [All BSD variants](https://enterprise.dejacode.com/licenses/?q=BSD) !! >;<